### DISCUSSION TITLE
Programming Language

### DISCUSSION THREAD

**Discussion by ansh027** [community/community#19891] [Other Feature Feedback, Questions, & Ideas] [answered]
Which language is good for begineer?

**Comment by Aryanstha** [comment, marked as answer]
When it comes to programming language for beginners, C is an excellent choice.

**Comment by felipezanardi** [comment]
Python is definitely better for beginners, so much easier to learn, to understand, and it's versatile.

**Comment by barrboss222** [comment]
Ru

**Comment by radar-blip** [comment]
Python. Bash scripts are fun and easy too, but not everyone considers that an actual programming language.

**Comment by Daniru2007** [comment]
I think python is great to get started with programming and to cover the basics of programming plus we can do a lot of things with python

**Comment by evan-hwang** [comment]
I don't know which country it is, but I think the language with the best books in the country is good. 
For example, Java is popular in Korea, so I recommend starting with Java because there are many good books or lectures.
